allensdk/brain_observatory/ecephys/ecephys_project_api/warehouse_patches/__init__.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
Is this warehouse patch meant to be a permanent fix or will the data be updated upstream?

Reply:
We don't have a release until March by which time I hope but cannot guarantee that it will have been fixed in the data.

I would like to have an automated check so that we don't accidentally continue applying this patch after the data are updated in March. I am concerned that until we actually apply the update the patch is not safe.

Reply:
In that case it would probably be a good idea to make a Github issue now about checking on/removing the patch so that it's documented in our backlog for March

Reply:
issue: #1076